app/views/layouts/application.html.haml

```diff
@@ -16,7 +16,7 @@
     = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.2/js/select2.full.min.js'
     = javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.3/js/i18n/sv.js'
     = javascript_include_tag Ckeditor.cdn_url
-    = javascript_include_tag "#{Rails.configuration.x.shf['google_maps_js_api']}?key=AIzaSyAhSCR7wvLZnInHGJu7XQ5PrPCyt3VDnU0"
+    = javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=#{Geocoder.config[:api_key]}"
```

Comment:
Did you want to keep the domain portion hard-coded for now?

I think we fixed the shf.yml and application.rb so that the domain could now be created via the SHF configuration.